hack/verify-conformance-requirements.sh

```diff
@@ -26,6 +26,11 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 source "${KUBE_ROOT}/hack/lib/util.sh"
 
+# Explicitly opt into go modules, even though we're inside a GOPATH directory
+export GO111MODULE=on
```

Comment:
we only do this when installing from hack/tools/go.mod. Kubernetes currently builds with go mod off, GOPATH mode on (and a fake local GOPATH), and uses vendor/.

this script should call `kube::golang::setup_env` instead (and that replaces `verify_go_version`, as it's a superset).
https://github.com/kubernetes/kubernetes/blob/770d3f181c5d7ed100d1ba43760a74093fc9d9ef/hack/lib/golang.sh#L500